### DISCUSSION TITLE
Taille de l'arène

### DISCUSSION THREAD

**Discussion by viannou** [Fouloscopie/robotique#2] [Q&A]
Bonjour Mehdi,

Tu indiques que l'arène a une taille de 3,2 x 1,6 mètres et que la cible est cachée dedans.

Or, dans le code, fichier `getTargetPosition.m` , on peut lire
```
    x_target = rand.*2.8 - 1.4 ;
    y_target = rand.*1.6 - 0.8 ;

```

Ce qui suggère une limite de recherche de la cible dans 2.,8 x 1,6 mètres...

J'imagine que tu corrigeras la zone de position possible de la cible ?...

**Comment by Fouloscopie** [comment]
Bonjour,

Bien vu ! Le code est correct mais il y a une erreur dans le descriptif du fichier README (c'est corrigé). 

L'arène fait **3,2 x 2** mètres. Mais les robots ne peuvent pas trop s'approcher des murs (pour éviter une collision). 
Pour éviter ce problème, je place la cible dans un rectangle intérieur qui fait 2,8 x 1,6 mètres (je garde 20 centimètres de sécurité de chaque coté). 

Merci !
